### DISCUSSION TITLE
zotero-style无法显示期刊和影响因子

### DISCUSSION THREAD

**Discussion by 13145699000** [MuiseDestiny/zotero-style#552] [Q&A]
![微信图片_20230809171146](https://github.com/MuiseDestiny/zotero-style/assets/141660614/a0907f7a-da1e-4acf-8c0d-91d2e0840f28)
![微信图片_20230809172527](https://github.com/MuiseDestiny/zotero-style/assets/141660614/0bb844cb-4d73-431a-bdd9-f18258ed7d68)
谢谢!

**Comment by MuiseDestiny** [comment]
你的插件不是最新版本。不提供支持

**Reply by 13145699000** [reply]
下载了最新版本也不行，已经试过很多个版本了，都不行欸

**Reply by MuiseDestiny** [reply]
你的期刊标签列设置请截图，截图不要拍照
